core/logs/src/main/scala/org/typelevel/otel4s/logs/LogRecordBuilder.scala

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright 2025 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.otel4s.logs
+
+import cats.Applicative
+import cats.Monad
+import org.typelevel.otel4s.AnyValue
+import org.typelevel.otel4s.Attribute
+import org.typelevel.otel4s.KindTransformer
+import org.typelevel.otel4s.meta.InstrumentMeta
+
+import java.time.Instant
+import scala.collection.immutable
+import scala.concurrent.duration.FiniteDuration
+
+/** Provides a way to build and emit a log record.
+  *
+  * @see
+  *   [[https://opentelemetry.io/docs/specs/otel/logs/api/#emit-a-logrecord]]
+  *
+  * @see
+  *   [[https://opentelemetry.io/docs/specs/otel/logs/data-model/]]
+  */
+trait LogRecordBuilder[F[_], Ctx] {
```

Comment:
That's the unfortunate part.

The spec explicitly says the API **must** accept the `Context`: 
> The [Context](https://opentelemetry.io/docs/specs/otel/context/) associated with the LogRecord. When implicit Context is supported, then this parameter SHOULD be optional and if unspecified then MUST use current Context. When only explicit Context is supported, this parameter SHOULD be required.